app/javascript/src/NewService/components/FormElements/FormWrapper.jsx

```diff
@@ -0,0 +1,31 @@
+// @flow
+
+import React from 'react'
+import type {FormProps} from 'NewService/types'
+
+const FormWrapper = (props: FormProps) => {
+  const {id, formActionPath, CSRFToken, HiddenServiceDiscoveryInput, submitText, ListItems, ListItemsProps} = props
```

Comment:
Since `CSRFtoken` doesn't keep a state and reads the meta props, it might not be needed to pass it as a param but import it and use it normally.